compiler/cgen.nim

```diff
@@ -1992,6 +2055,8 @@ proc updateCachedModule(m: BModule) =
 proc myClose(graph: ModuleGraph; b: PPassContext, n: PNode): PNode =
   result = n
   if b == nil: return
+  if n == nil:
```

Comment:
simply `doAssert n!=nil`

Reply:
I don't use `doAssert` because I don't understand it.